chain/jsonrpc/res/last_blocks.html

```diff
@@ -32,9 +32,10 @@
     </style>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script>
-        function createHashElement(hashValue, creator) {
+        function createHashElement(hashValue, creator, className) {
             var element = $('<span>');
             element.addClass('hashElem');
+            element.addClass(className);
             element.attr("hashValue", hashValue);
```

Comment:
`"hashvalue"` (or alternatively use `"hashValue"` where you read the value).

Reply:
done